Batch modify v5.1 docs (#254)
tikv/website: content/docs/5.1/reference/architecture/terminology.md

---
title: Terminologies
description: Explain TiKV and PD terminologies.
menu:
    "5.1":
        parent: Architecture
        weight: 4
---

This document explains some important terminologies of TiKV and PD.

## Node

A TiKV **Node** is a physical node in the cluster, which might be a virtual machine, a container, etc. Within each Node, there can be one or more stores.

> The status for the Node, Store, and Region is regularly reported to the Placement Driver.

Reviewer: ```suggestion
The Node, Store, and Region regularly report their status to the Placement Driver.
```
